src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -186,23 +199,35 @@ uct_ib_mlx5_devx_reg_ksm_data_addr(uct_ib_mlx5_md_t *md, struct ibv_mr *mr,
                                    struct mlx5dv_devx_obj **mr_p,
                                    uint32_t *mkey)
 {
-    int i;
-    char *in;
-    void *klm;
     ucs_status_t status;
+    uint64_t mr_address;
+    void *klm;
+    char *in;
+    int i;
+
+    if (reason != NULL) {
+        ucs_trace("%s: initialize ksm_data_addr list for %s address 0x%lx "
+                  "length %zu iova 0x%" PRIx64,
+                  uct_ib_device_name(&md->super.dev), reason, address, length,
+                  iova);
+    }
```

Comment:
minor: can be inline func and reused also in `uct_ib_mlx5_devx_reg_ksm_data_mt` (together with check `reason != NULL`)